src/modules/shared/components/Footer/Footer.constants.ts

```diff
@@ -0,0 +1,5 @@
+export enum footerType {
```

Comment:
Ik denk niet dat deze nodig gaan zijn. Zie naar de andere feedback in `Footer.tsx`. 
Het probleem dat ik hierbij zie is wat als je bv. de feedback wil combineren met de "simple" variant?